src/components/Button/index.tsx

```diff
@@ -1,13 +1,76 @@
-import React, { forwardRef } from 'react';
+import React, { forwardRef, HTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
 import cc from 'classcat';
 import baseElement from 'util/baseElement';
 import { CaretDownIcon } from 'lib/icons/react';
 
-export const SIZES = ['s', 'm'];
-export const THEMES = ['blue', 'orange', 'outlined', 'outlined--blue', 'ghost'];
+export const SIZES = ['s', 'm'] as const;
 
-const Button = forwardRef(
+export const THEMES = ['blue', 'orange', 'outlined', 'outlined--blue', 'ghost'] as const;
+export type THEME = typeof THEMES[number];
```

Comment:
Stylistically, I think this should be title-cased:

```suggestion
export type Theme = typeof THEMES[number];
```